x-pack/test/encrypted_saved_objects_api_integration/tests/encrypted_saved_objects_api.ts

```diff
@@ -900,5 +902,79 @@ export default function ({ getService }: FtrProviderContext) {
         }
       });
     });
+
+    describe('enforceRandomId', () => {
```

Comment:
Nit:
For readability, would restructuring this allow us to be more concise?

```
describe('enforceRandomId', () => {
  describe('enforceRandomId false', () => {
    it('#create allows setting non-random ID', async () => {
      ...
    }
    it('#bulkCreate allows setting non-random ID', async () => {
      ...
    }
  }
  describe('enforceRandomId default true', () => {
    it('#create disallows setting non-random ID', async () => {
      ...
    }
    it('#bulkCreate disallows setting non-random ID', async () => {
      ...
    }
  }
}
```

Reply:
Addressed in https://github.com/elastic/kibana/pull/198287/commits/67ca7d00e3a2d7438fc093246ce6eae28b5b386f#diff-b162c94eeb50773c3302e40ff8e9487d1c9b751b6504d89a12ae0784b426355fR907